src/commands/cli/tarballs/smoke.ts

```diff
@@ -68,7 +69,6 @@ export default class SmokeTest extends SfdxCommand {
   }
 
   private getAllCommands(executable: string): string[] {
-    if (path.basename(executable) === CLI.SF) this.execute(executable, 'plugins:install @oclif/plugin-commands@^2');
```

Comment:
the sign verification hook only supports passing a specific version or dist-tag as a specifier. Logged `W-11771849` to fix that.